pkg/action/custom_resource_lister.go

```diff
@@ -0,0 +1,138 @@
+package action
+
+import (
+	"context"
+	"fmt"
+	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
+	"k8s.io/apimachinery/pkg/runtime/schema"
+
+	"github.com/operator-framework/api/pkg/operators/v1alpha1"
+	"github.com/operator-framework/api/pkg/operators/v2alpha1"
+	apiextensionsv1 "k8s.io/apiextensions-apiserver/pkg/apis/apiextensions/v1"
+	k8serrors "k8s.io/apimachinery/pkg/api/errors"
+	"k8s.io/apimachinery/pkg/types"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+)
+
```

Comment:
I would suggest:
1. Combining ` internal/pkg/action/operator_list_custom_resources.go` and this file.
2. Putting the combined file at `pkg/action/operator_list_operands.go` so that it can be imported for external client's uses.
3. Unexporting everything except `OperatorListOperands` and its `Run` function.
4. Moving `internal/pkg/action/config.go` to `pkg/action/config.go`

Reply:
ok,done

Reply:
pulling out the `Configuration` to its standalone file is producing a pretty big diff so right now I'm just going to import the existing struct and we can pull it out in another PR or commit

Reply:
Sounds good. We just have to do that before external projects can use it because `action.Configuration` is needed when constructing the action struct.